src/aliases.js

```diff
@@ -83,7 +84,7 @@ const getJestAliases = aliasHandler => Object.entries(state.aliases).reduce(
       return aliasHandler(aliasMap, key, path)
 
     const formattedKey = key + '(.*)'
-    aliasMap[formattedKey] = path + '$1'
+    aliasMap[formattedKey] = pathJoin(path, '$1')
```

Comment:
Here it appends a path separator to the aliasMap for the formatted key to match what is documented here: https://github.com/ilearnio/module-alias#usage-with-jest